### PR TITLE
New Theme

### PR DESCRIPTION
**Pull request by seanmarcia**
### What github issue is this PR for, if any?
Resolves #2XXX

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -22,6 +22,14 @@ def default_page_header
     "CASA / Volunteer Tracking"
   end
 
+  def session_signout_link
+    if user_signed_in?
+      destroy_user_session_path
+      elsif all_casa_admin_signed_in?
```
Comment (compwron) [inline app/helpers/application_helper.rb:28]: the linter might dislike this

Comment (seanmarcia) [reply app/helpers/application_helper.rb:28]: Definitely. This was mostly to get the sign out working.
There will need to be some refactor issues for this, the `active_class` helper method, the `menu_item` helper method and a bunch more. Also how the filtering works, and more :)

Comment (seanmarcia) [reply app/helpers/application_helper.rb:28]: Also, please pull and re-test. I didn't realize that the /public/assets director was in the gitignore file.

**Comment by FireLemons** [comment]
wait why is this closed

**Pull request closed without merging**